app/src/main/java/com/example/simplenavigation/MainActivity.java

```diff
@@ -46,7 +46,7 @@ public void onClick(View view) {
                 float yaw = Float.parseFloat(editTextYaw.getText().toString());
                 int tilt = Integer.parseInt(editTextTilt.getText().toString());
                 mRobot.goToPosition(new Position(posX, posY, yaw, tilt));
-                mRobot.tiltAngle(tilt,1);
+                mRobot.tiltBy(tilt);
```

Comment:
In this case, use `tiltAngle`, not `tiltBy`. The user will input an absolute angle, not a relative angle.

Reply:
`mRobot.tiltAngle(tilt,1);` wasn't working, robot arrives at set destination but irl display tilt angle doesn't correspond to set value so I wanted to try use `mRobot.tiltBy(tilt);` instead to see if it will solve the issue or not but it still behaved the same way.

Code changed back to `mRobot.tiltAngle(tilt,1);`